src/core/transaction.js

```diff
@@ -1112,6 +1112,77 @@ Transaction.prototype.setDeliverMin = function(deliver_min) {
   return this._setAmount('DeliverMin', deliver_min);
 };
 
+Transaction.prototype.suspendedPayment = function(options_) {
+  const options = _.merge({}, options_);
+
+  if (_.isUndefined(options.cancel_after)) {
+    options.cancel_after = options.cancelAfter;
+  }
+  if (_.isUndefined(options.finish_after)) {
+    options.finish_after = options.finishAfter;
+  }
+
+  this.setType('SuspendedPaymentCreate');
+  this.setAccount(options.account);
+  this.setDestination(options.destination);
+  this.setAmount(options.amount);
+
+  if (!_.isUndefined(options.digest)) {
+    this._setHash256('Digest', options.digest);
+  }
+  if (!_.isUndefined(options.cancel_after)) {
+    this.setCancelAfter(options.cancel_after);
+  }
+  if (!_.isUndefined(options.finish_after)) {
+    this.setFinishAfter(options.finish_after);
+  }
+
+  return this;
+};
+
+Transaction.prototype.setCancelAfter = function(cancel_after) {
+  return this._setUInt32('CancelAfter', cancel_after);
+};
+Transaction.prototype.setFinishAfter = function(finish_after) {
+  return this._setUInt32('FinishAfter', finish_after);
+};
+
+Transaction.prototype.suspendedPaymentCancel = function(options_) {
```

Comment:
I guess this func is a better candiate for default args feature. 

We don't seem to be consistently allowing camelCase or snake_case though? See `offer_sequence`